DataVisualisation01.pv

```diff
@@ -34,10 +34,18 @@ for bird in species_nameData:
             x = xOfDot(float(bird['lon']))
             y = yOfDot(float(bird['lat']))
             for i, c in enumerate(bird["colors"]):
-                fill(c["r"], c["g"], c["b"], 0.5)
-                x_with_offset = x + 5
-                y_with_offset = y + -5
-                diameter = 40.0
+                r = c["r"] / 255.0
```

Comment:
We have to convert the color value from the `0 - 255` range into a `0.0 - 1.0` range